galite-core/src/main/kotlin/org/kopi/galite/visual/form/VBlock.kt

```diff
@@ -983,7 +982,8 @@ abstract class VBlock(var title: String,
       }
       if (!fields[index].hasAction() &&
           fields[index].getAccess(activeRecord) >= VConstants.ACS_VISIT &&
-          (isDetailMode && !fields[index].noDetail() || !isDetailMode && !fields[index].noChart())) {
+          (isDetailMode && !fields[index].noDetail() || !isDetailMode && !fields[index].noChart())
+          && fields[index] !is VImageField) {
```

Comment:
What's the point of this change? What does it do? Same question for the rest of the changes in this class.